debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/SharedClassesMetaDataHelper.java

```diff
@@ -23,14 +23,20 @@
 
 import com.ibm.j9ddr.CorruptDataException;
 import com.ibm.j9ddr.vm29.pointer.generated.J9ShrOffsetPointer;
+import com.ibm.j9ddr.vm29.types.UDATA;
+import java.lang.reflect.*;
 
 public class SharedClassesMetaDataHelper {
 	public static int getCacheLayerFromJ9shrOffset(J9ShrOffsetPointer j9shrOffset) throws CorruptDataException {
 		int layer;
 		try {
-			layer = j9shrOffset.cacheLayer().intValue();
-		} catch (NoSuchFieldError | NoSuchMethodError e) {
+			Method m = j9shrOffset.getClass().getMethod("cacheLayer");
```

Comment:
Is this method called frequently enough that it should cache the method?

Reply:
Remove `Method m`.